src/frontend/app/features/cloud-foundry/cloud-foundry-tabs-base/cloud-foundry-tabs-base.component.html

```diff
@@ -4,7 +4,7 @@ <h1>{{ (cfEndpointService.endpoint$ | async)?.entity.name }}</h1>
     <button mat-icon-button name="add-org" *ngIf="(this.cfEndpointService.connected$ | async)" routerLink="/cloud-foundry/{{this.cfEndpointService.cfGuid}}/add-org">
```

Comment:
This file and a few others don't seem to relate to the about page and looks like its form a PR I did that was merged... you probably need to rebase with v2-master.